src/gretel_synthetics/config.py

```diff
@@ -114,7 +130,10 @@ class BaseConfig:
 
     # Training configurations
     max_lines: int = 0
-    epochs: int = 15
+    epochs: int = 250
```

Comment:
Does not match the default value described in the docstring above (150). I'd recommend 100 on the top-end as a setting for epochs as you'll likely be overfitting the model at that point